modules/integration/src/test/scala/scala/cli/integration/SparkTestDefinitions.scala

```diff
@@ -129,7 +129,7 @@ abstract class SparkTestDefinitions(val scalaVersionOpt: Option[String]) extends
       val extraEnv =
         if (needsWinUtils) maybeHadoopHomeForWinutils(root / "hadoop-home")
         else Map.empty[String, String]
-      val res = os.proc(TestUtil.cli, "run", extraOptions, "--spark-standalone", "--jvm", "8", ".")
+      val res = os.proc(TestUtil.cli, "run", extraOptions, "--spark-standalone", ".")
```

Comment:
Does the default of scala-cli use java 8 even if I have on path java 17? If yes, I think it should be warning what happened.

Reply:
Yeah, if the system JVM is at least 8, maybe we should use it instead… 🤔

Reply:
I'll give it a try in another PR.